tests/command.functions.test.js

```diff
@@ -242,7 +240,7 @@ test('should use settings from netlify.toml dev', async (t) => {
       })
       .buildAsync()
 
-    await withFunctionsServer({ builder }, async () => {
+    await withFunctionsServer({ builder, port }, async () => {
```

Comment:
This was the bug in the test that was causing it to be unstable.

The CLI was using a custom port from `netlify.toml` but the test was waiting for the default port to be open.
It worked sometimes since we run the tests in parallel and we have a test that opens the default port.